ynput/OpenPype
# Houdini: Change node type for OpenPypeContext `null` -> `subnet`

## Changelog Description

Change the node type for OpenPype's hidden context node in Houdini from `null` to `subnet`. This fixes #4734 

## Additional info

This makes it so that for example a plug-in like Redshift doesn't try and add its own spare parameters on the node bloating the size of the context node. This way it also doesn't affect how we're reading the data from the node (which is by retrieving all spare parameters).

Setting `run_init_scripts=False` also avoids e.g. plug-in initialization scripts for a node type to try and add spare parameters too.

## Testing notes:

1. Start Houdini
2. Run publisher in a new scene
3. You should now have a hidden OpenPypeContext node of type 'subnet'
4. Open an older scene that used the new publisher - it will preserve the old `null` type and should not break anything.

You can see the hidden node in the "Tree View" by right clicking on the top bar of the tree view UI and then enable "Show Hidden".
![afbeelding](https://user-images.githubusercontent.com/2439881/228670744-db71f428-0548-4483-9289-c02006bbc868.png)
